MarkBind/markbind
# Remove variable syntax for icons

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [x] Other, please explain: depreciation of old feature.

``` Markdown
**Note: This is a breaking change, will have to increment a minor version for next release
```


Resolves #712 



**What is the rationale for this request?**

Since #680 syntax change, the team has postponed depreciating the old syntax until the academic semester is over. Now that it is summer, the team has the green light to proceed with the depreciation.

**What changes did you make? (Give an overview)**
- Remove `glyphicon.csv` and `font-awesome.csv` from `src/assets`
- Remove `getIconsMap` method from `Site.js`, along with any related constants used.
- Update `Site.test.js` unit tests
- Fix minor documentation nit within `icons.mbdf`

**Is there anything you'd like reviewers to focus on?**
- Commit messages

**Testing instructions:**
1. Open Netlify Preview and check that icons within the documentation is not broken.
2. Checkout the PR branch, Init, author a new MarkBind website with the old syntax for icons e.g. `{{ glyphicon-home }}`
3. The icons should be rendered as `NaN` as it doesn't exist.
4. Replace the double braces `{{ }}` with colons `: :` and the icons should render properly.
